entity-framework/core/modeling/relationships.md

```diff
@@ -12,15 +12,15 @@ A relationship defines how two entities relate to each other. In a relational da
 > [!NOTE]  
 > Most of the samples in this article use a one-to-many relationship to demonstrate concepts. For examples of one-to-one and many-to-many relationships see the [Other Relationship Patterns](#other-relationship-patterns) section at the end of the article.
 
-## Definition of Terms
+## Definition of terms
 
 There are a number of terms used to describe relationships
 
 * **Dependent entity:** This is the entity that contains the foreign key property(s). Sometimes referred to as the 'child' of the relationship.
 
 * **Principal entity:** This is the entity that contains the primary/alternate key property(s). Sometimes referred to as the 'parent' of the relationship.
 
-* **Foreign key:** The property(s) in the dependent entity that is used to store the values of the principal key property that the entity is related to.
```

Comment:
Should this be "_are_ used"?

Reply:
Argh, this is cause by "property(s)", I'll just convert everything to plural